contracts/src/token/erc721/extensions/enumerable.rs

```diff
@@ -144,6 +149,13 @@ impl IErc721Enumerable for Erc721Enumerable {
     }
 }
 
+impl IErc165 for Erc721Enumerable {
```

Comment:
I am not sure if we want to have it implemented always.

Reply:
Basically it is [implemented](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/530179a71f435e85ae9df9b9f12b5637cf229e5c/contracts/token/ERC721/extensions/ERC721Enumerable.sol#L39) for Erc721Enumerable in solidity